components/kubeflow/katib-launcher/src/launch_experiment.py

```diff
@@ -41,6 +41,9 @@ def __init__(self, version="v1alpha3", client=None):
     super(Experiment, self).__init__(ExperimentGroup, ExperimentPlural, version, client)
 
   def is_expected_conditions(self, inst, expected_conditions):
+    trials_running_count = inst.get('status', {}).get('trialsRunning', 0)
+    if 0 != trials_running_count:
```

Comment:
If you skip the default on the second `get` you could do this the same way as on row 48 I believe to keep it more consistent, by default `get` will return None